PWGHF/vertexingHF/AliAnalysisTaskSEXic2eleXifromAODtracks.cxx

```diff
@@ -3142,7 +3139,7 @@ void AliAnalysisTaskSEXic2eleXifromAODtracks::DefineCascTreeVariables()
   fCandidateVariableNames[25]="EvNumber";
 
   for (Int_t ivar=0; ivar<nVar; ivar++) {
-    fCascVariablesTree->Branch(fCandidateVariableNames[ivar].Data(),&fCandidateCascVariables[ivar],Form("%s/F",fCandidateVariableNames[ivar].Data()));
+    fCascVariablesTree->Branch(fCandidateVariableNames[ivar].Data(),&fCandidateCascVariables[ivar],Form("%s/f",fCandidateVariableNames[ivar].Data()));
```

Comment:
With this you fill the tree with `Float16_t` instead of `Float_t`, is is intended?

Reply:
Hi, Fabrizio. Yes, I remember it is  because of the ROOT version.

Reply:
Hi Tiantian, yes in the old ROOT versions `f` corresponded to `Float_t`, now `F` is the argument for `Float_t`. However now you are changing it to `f` that in recent ROOT versions corresponds to `Float16_t` (hence less precision, since `Float16_t` are 24 bit floating point with truncated mantissa instead of the usual 32 bits, see https://root.cern.ch/doc/master/classTTree.html). Did you want to reduce the precision of your variables to reduce the dimension of your tree?

Reply:
yes, you are right. Sorry, just now I did not read it carefully. It should be "F".  I remember I did the commit (changed 'f' to 'F')few days ago, it is strange that it is back again. It could be due to my git has a problem.  I will push again. Thanks for your check.

Reply:
The new version was committed just now. I was using the code version that the one before I made 'f' to 'F'. That's why the 'f' to 'F' was back again... sorry for the mistake